Source/ablastr/fields/PoissonSolver.H

```diff
@@ -121,7 +127,8 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
     amrex::Real max_norm_b = 0.0;
     for (int lev=0; lev<=finest_level; lev++) {
         using namespace ablastr::constant::SI;
-        rho[lev]->mult(-1._rt/ep0);  // TODO: when do we "un-multiply" this? We need to document this side-effect!
+
```

Comment:
unintentional change?
```suggestion
```

Reply:
Is this about the empty line? If so, yes